docs/components/ar-hit-test.md

```diff
@@ -0,0 +1,48 @@
+---
+title: hit-test
+type: components
+layout: docs
+parent_section: components
+source_code: src/components/hit-test.js
+examples: []
+---
+
+[hit-test]: https://immersive-web.github.io/hit-test/
+
+This component uses the WebXR hit-test API to position virtual objects in the real world. Remember to request the `hit-test` optional feature to allow it work.
+
+You add it to the scene element and then when the user is in Augmented Reality if they tap on the screen or select with a controller it will work out where the user is pointing to in the real world and place a reticle there and fire events.
+
+If you have set a target property to be an element it will automatically make the reticle to be the same size as the footprint of the target and then when the user selects the object will be telepoted there. It will also set the visible state of the object to true so you can hide the object until the user has placed it for the first time. The `hide-on-enter-ar` component is useful for that.
+
+You can toggle this component's enabled state to not do any interactions until you are ready.
+
+## Example
+
+```html
+<a-scene webxr="optionalFeatures:  hit-test;" ar-hit-test="target:#myobject;">
+	<a-entity id="myobject"></a-entity>
+```
+
+## Properties
+
+| Property       | Description                                                  | Default Value |
+|----------------|--------------------------------------------------------------|---------------|
+| target         | The object to move around.                                   | null          |
+| enabled        | Whether to do hit-testing or not                             | true          |
+| map            | Image to use for the reticle                                 | ""            |
```

Comment:
I believe this is called `src` in the code. Do you think we should expose it now or keep it non-configurable for simplicity until the need arise?

Reply:
Nice catch i'll update the doc.

I would like to keep it in case people want to replace it with something branded or product specific such as an image with dimensions.